<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerModifyDialog</class>
 <widget class="QDialog" name="WorkerModifyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>577</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>577</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 154, 38);
background-image: url(:/resource/photo/Resource/2.jpg);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;仿体&quot;;</string>
   </property>
   <property name="text">
    <string>输入修改需要的项，若不输入则不变：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>281</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;仿体&quot;;</string>
        </property>
        <property name="text">
         <string>名    字：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;仿体&quot;;</string>
        </property>
        <property name="text">
         <string>部    门：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_departmen">
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;仿体&quot;;</string>
        </property>
        <property name="text">
         <string>岗    位：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_position">
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;仿体&quot;;</string>
        </property>
        <property name="text">
         <string>地    址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_address">
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_Modify">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;仿体&quot;;</string>
        </property>
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_back">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;仿体&quot;;</string>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;新宋体&quot;;
</string>
   </property>
   <property name="text">
    <string>工资管理系统</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_address</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_position</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_departmen</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
